packages/blade/src/components/Checkbox/CheckboxGroup/CheckboxGroup.tsx

```diff
@@ -56,7 +56,7 @@ type CheckboxGroupProps = {
   /**
    * Renders the label of the checkbox group
    */
-  label: string;
+  label?: string;
```

Comment:
i'm more concerned with the scary patterns emerging with this 😢 people not using label because they want to use a large label 😭